client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -66,21 +66,25 @@ export class CartItem extends React.Component {
 		}
 
 		if ( isGSuiteProductSlug( cartItem.product_slug ) ) {
-			const { cost_before_coupon: costBeforeCoupon } = cartItem;
+			const {
+				cost_before_coupon: costBeforeCoupon,
+				is_sale_coupon_applied: isSaleCouponApplied,
+			} = cartItem;
+			const { is_coupon_applied: isCouponApplied } = cart;
 
-			if ( costBeforeCoupon ) {
+			if ( isSaleCouponApplied ) {
 				return (
 					<div className="cart__gsuite-discount">
-						<span className="cart__gsuite-discount-regular-price">
-							{ costBeforeCoupon }
-						</span>
+						<span className="cart__gsuite-discount-regular-price">{ costBeforeCoupon }</span>
 
 						<span className="cart__gsuite-discount-discounted-price">
 							{ cost } { cartItem.currency }
 						</span>
 
 						<span className="cart__gsuite-discount-text">
-							{ translate( 'Discount for first year' ) }
+							{ isCouponApplied
+								? translate( 'Multiple discounts' )
```

Comment:
I think we could also simply display nothing when both a coupon code (i.e. a coupon entered by the user on the checkout page) as well as a sale coupon (i.e. a coupon automatically applied to a product such as the two-month discount) apply. @fditrapani, what do you think?

Reply:
It would still be helpful to show something. "Discounts applied"  would be a better way to word it.

Reply:
@fditrapani: we don't display anything when the user enters a coupon code for other products (actually we don't even cross the corresponding costs out). Do you still think we should display `Discounts applied` even if it's not consistent?

No coupon code | With coupon code
------ | -----
![screenshot](https://user-images.githubusercontent.com/594356/70433853-6d045c00-1a8b-11ea-8d2c-39428367ba3f.png) | ![screenshot](https://user-images.githubusercontent.com/594356/70433940-9fae5480-1a8b-11ea-97d4-e03b6cc6c992.png)